website/docs/docs/dbt-cloud-apis/schema-discovery-modelByEnv.mdx

```diff
@@ -6,7 +6,13 @@ id: "discovery-schema-modelByEnv"
 
 import { ArgsTable, SchemaTable } from "./schema";
 
-<Snippet path="discovery-public-preview-banner" />
+:::caution
+
+dbt Labs is making changes to the Discovery API to support multi-cell environment. These changes will take effect on September 7, 2023. 
+
+The model by environment object is being deprecated and will be replaced with `environment { applied { modelHistoricalRuns } }`. You will need to update your API calls accordingly to avoid errors. Refer to the 
```

Comment:
```suggestion
The model by environment object is being deprecated and will be replaced with `environment { applied { modelHistoricalRuns } }`. When the time comes, you will need to update your API calls accordingly to avoid errors. Refer to the 
```